frontend/src/components/Home/Autocomplete.tsx

```diff
@@ -179,7 +185,16 @@ export default function Autocomplete() {
         placeholder="Search by landlord or building address"
         className={text}
         variant="outlined"
-        onKeyDown={(event) => (event.key === 'ArrowDown' ? setFocus(true) : setFocus(false))}
+        onKeyDown={(event) => {
+          if (event.key === 'ArrowDown') {
+            setFocus(true);
+          } else if (event.key === 'Enter') {
+            setFocus(true);
+            history.push('/searchresults');
```

Comment:
So usually what happens when you search is that the URL will show the query parameters (the words after the `?`)- see the google image below. The query param is `q` and the value is `cats`. This way, a person can directly navigate to a page of search results, and this is more standard in industry. 
<img width="325" alt="Screen Shot 2022-03-20 at 4 13 29 PM" src="https://user-images.githubusercontent.com/57203639/159184189-a97949ff-6043-4e17-bff9-44ae0c528049.png">

How about we change `history.push('/searchresults');` to ```navigate(`/search?q=${query}`);```? Then the URL will show the query that the user types into the search bar